backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5096,4 +5096,34 @@ databaseChangeLog:
               - column:
                   name: equipment_uid_type
                   type: text
-                  remarks: Equipment UID from LIVD sync to be used in the universal pipeline
\ No newline at end of file
+                  remarks: Equipment UID from LIVD sync to be used in the universal pipeline
+  - changeSet:
+      id: add-disease_and_destination-columns-to-upload
+      author: tsv7@cdc.gov
+      comment: Add disease and destination(COVID|UNIVERSAL) columns to upload table
+      changes:
+        - tagDatabase:
+            tag: add-disease_and_destination-columns-to-upload
+        - sql:
+            remarks: Create the enumerations needed for pipeline types.
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.PIPELINE as ENUM('COVID', 'UNIVERSAL');
+        - addColumn:
+            tableName: upload
+            columns:
+              - column:
+                  name: destination
+                  type: ${database.defaultSchemaName}.PIPELINE
+                  remarks: the pipeline where the results were sent to
+              - column:
+                  name: supported_disease_id
+                  remarks: The disease that was reported.
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__upload__supported_disease_id
+                    referencedTableName: supported_disease
+                    referencedColumnNames: internal_id
+      rollback:
+        sql: |
```

Comment:
in the rollback don't we want to drop the `supported_disease_id` column? 🤔

Reply:
My understanding is that [addColumn](https://docs.liquibase.com/change-types/add-column.html) has rollback automatically performed. But would love @BobanL to confirm this!

Reply:
I rolledback locally and saw the `destination` column get dropped but the `supported_disease_id` remained -- not sure if anyone else can repro

Reply:
let me try again as a sanity check 😅

Reply:
ah actually i do see the second `supported_disease_id` still after the rollback

Reply:
wonder if it is due to the constraints... I will add the drop column in the rollback section. Thanks for catching that @emyl3 !

Reply:
I wonder if you can't mix and match auto rollback vs defining your own rollback https://docs.liquibase.com/workflows/liquibase-community/using-rollback.html?_ga=2.72264851.1204933114.1698177543-1193148286.1698177543 (see "Override default rollback commands" section) 🤔